aws/resource_aws_glue_catalog_database_test.go

```diff
@@ -208,6 +257,21 @@ resource "aws_glue_catalog_database" "test" {
 `, rInt, desc)
 }
 
+func testAccGlueCatalogDatabase_validation(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_glue_catalog_database" "test" {
+  name = "%s"
+  description = "validation test"
```

Comment:
Given the rest of the arguments are optional and we're only looking to test the `name` attribute, I'd suggest removing the other arguments from the test configuration.